docs/GraFx-Studio/concepts/design-run/index.md

```diff
@@ -0,0 +1,63 @@
+# Design Mode and Run Mode
+
+GraFx Studio provides two modes for creating and testing Smart Templates: **Design Mode** and **Run Mode**. 
+
+These modes allow you to focus separately on designing the smart template and testing business logic.
+
+In **Design Mode**, you can freely arrange and style elements without triggering actions or interactions. This keeps the workspace clean, ideal for setting up and adjusting layouts, styles, and initial logic.
```

Comment:
"without triggering actions or interactions" -> What do you mean with interactions? I would rather say "without triggering actions that can mess up your original design"